debug/main.py

```diff
@@ -38,7 +38,7 @@ def bgp():
         """debug bgp group """
         pass
 
-    @bgp.command()
+    @bgp.command('allow_martians')
```

Comment:
allow_martians -> allow-martians

Reply:
Changed. Also changed in undebug/main.py.